cpp/python/src/future.cpp

```diff
@@ -60,13 +60,18 @@ static PyObject* get_asyncio_future_object()
   }
 
   asyncio_module = PyImport_Import(asyncio_str);
-  if (PyErr_Occurred()) ucxx_trace_req("Python error here");
+  if (PyErr_Occurred()) {
+    ucxx_trace_req("Error importing asyncio");
```

Comment:
Would it make sense to include the function name here?


```suggestion
    ucxx_trace_req("ucxx::python::get_asyncio_future_object, error importing asyncio");
```

Same question for other added logging here

Reply:
I had intentionally omitted those here because messages in those cases are unique enough they can't be misinterpreted. But yeah, I agree now we should do that to keep things more consistent. Fixed in https://github.com/rapidsai/ucxx/pull/156/commits/f9fa937fc23b34e9aa179b8727fe93881ee763e1 .